src/app.ts

```diff
@@ -0,0 +1,67 @@
+import * as http from "http";
+import {IncomingMessage, Server, ServerResponse} from "http";
+import {ListenOptions} from "net";
+import * as fs from "fs";
+
+const filePath: string = './text/exampleFile.txt';
+const contentOptions: ListenOptions = {
+    host: 'localhost',
+    port: 3000
+};
+
+const server: Server = http.createServer((req: IncomingMessage, res: ServerResponse) => {
+    res.statusCode = 200;
+    res.setHeader('Content-Type', 'text/plain');
+    dataByUrl(req, res);
+});
+
+server.listen(contentOptions, () => {
+    console.log(`Server running at http://${contentOptions.host}:${contentOptions.port}/`);
+});
+
+function dataByUrl(req: IncomingMessage, res: ServerResponse) {
+    if (req.url === '/CONTENT') {
+        content(res);
+    }
+    else if (req.url === '/updateTime') {
+        updateTime().then(data => {
+            res.statusCode = 200;
+            res.write(data);
+            res.end();
+        }).catch(err => {
+            res.statusCode = 404;
```

Comment:
res.status(404).send(err.message).
404 isnt the best http status here  ... send different status